moon-store-mac-os/moon-store-mac-os/Networking/Models/UserRegisterModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UserRegisterModel.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 1/7/25.
+//
+
+struct UserRegisterModel: Codable {
+    var firstName: String = ""
+    var lastName: String = ""
+    var identification: String = ""
+    var phone: String = ""
+    var address: String = ""
+    var email: String = ""
+    var password: String = ""
+    var confirmPassword: String = ""
+    
+    func cannotRegisterYet() -> Bool {
+        firstName.isEmpty ||
+        lastName.isEmpty ||
+        identification.isEmpty ||
+        phone.isEmpty ||
+        address.isEmpty ||
+        !email.matchesEmail ||
+        password.isEmpty ||
+        confirmPassword.isEmpty
```

Comment:
Me parecio ver el metodo `isNotEmpty` como extension, en caso de que no exista, solamente usa `!firstName.isEmpty`

```suggestion
        firstName.isNotEmpty &&
        lastName.isNotEmpty &&
        identification.isNotEmpty &&
        phone.isNotEmpty &&
        address.isNotEmpty &&
        email.matchesEmail &&
        password.isNotEmpty &&
        confirmPassword.isNotEmpty
```

Reply:
Esto haria cambiar la logica y nombre de la variable.